index.html

```diff
@@ -67,5 +68,193 @@ <h1 class="social-media-heading">LET’S CONNECT</h1>
         </div>
       </section>
     </main>
+    <!-- The works pages -->
+    <section class="grid-container">
+      <section class="works">
+        <div class="tonic-1st">
+          <img class="tonic-1st-img" src="images/Snapshoot Portfolio.png" />
+          <h1 class="tonic-1st-heading">Tonic</h1>
+          <ul class="tonic-1st-list">
+            <li class="canopy">CANOPY</li>
+            <li class="list-vector-icon">
+              <i class="fa-solid fa-circle-small"></i>
+            </li>
+            <li class="backend">Back End Dev</li>
+            <li class="list-vector-icon">
+              <i class="fa-solid fa-circle-small"></i>
+            </li>
+            <li class="year">2015</li>
+          </ul>
+          <p class="works-passage">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="works-btns">
+            <li><button type="button">html</button></li>
+            <li class="works-btn"><button type="button">css</button></li>
+            <li class="works-btn">
+              <button type="button">javascript</button>
+            </li>
+          </ul>
+          <button class="see-project-btn" type="button">See Project</button>
```

Comment:
- [ ] Also, kindly add button interaction (active, hover) to all your button elements, this will enhance a better user interaction. 👍